client/src/components/Notifications/Notification.tsx

```diff
@@ -1,4 +1,45 @@
+import { theme } from "consts/theme";
 import React, { useEffect, useState } from "react";
+import styled, { StyledComponent } from "styled-components";
+
+const Notifications = styled.div`
+  will-change: transform;
+  font-size: 1rem;
+  overflow: hidden;
+  background-color: ${theme.notifications.backgroundColor};
```

Comment:
```suggestion
  background-color: ${({ theme }) => theme.notifications.backgroundColor};
```

change the static value into a function that returns it to ensure reactivity when switching theme...

Let's do it across this PR if there are more instances of theme usage...